### PR TITLE
OPS-3102 Added infra-schulcloud-01 and infra-schulcloud-02

### PR DESCRIPTION
**Pull request by simoncolincap**
# Description
Added infra stage group and two instances in that group infra-schulcloud-01 and infra-schulcloud-02

## Links to Tickets or other pull requests
https://ticketsystem.hpi-schul-cloud.org/browse/OPS-3102

## Approval for review

- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definiton of Done

More and detailed information on the _definition of done_ can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

File: .github/workflows/rollout.yml [hunk 1]
```diff
@@ -250,3 +250,31 @@ jobs:
...
+      host_group: default
+      host_stage: infra
+      execute_group: ${{ github.event.inputs.instance_group }}
+    secrets:
+      token: ${{ secrets.GITHUB_TOKEN }}
+      ONEPASSWORD_VAULT: ${{ secrets.ONEPASSWORD_VAULT }}
+      ETHERPAD_API_KEY: ${{ secrets.ETHERPAD_API_KEY }}
+      KUBECONFIG: ${{ secrets.KUBECONFIG }}
+
+  infra-schulcloud-02:
```
Comment (mamutmk5) [inline .github/workflows/rollout.yml:268]: please put it behind the Approval stage approve_ref
```suggestion
  infra-schulcloud-02:
    needs:
      - approve_ref 
```

File: ansible/hosts [hunk 2]
```diff
@@ -37,6 +39,13 @@ test-loadtest-01
 [test:vars]
 ansible_group_priority=10
 
+[infra]
+infra.schulcloud-01
+infra.schulcloud-02
+
+[infra:vars]
+ansible_group_priority=10
+
```
Comment (mamutmk5) [inline ansible/hosts:48]: What is the differnce in the config of the group_vars between infra and test?

**Review by mamutmk5 — CHANGES_REQUESTED** [review]

File: .github/workflows/rollout.yml (hunk 1 above)
Comment (simoncolincap) [reply .github/workflows/rollout.yml:268]: The infra-schulcloud instances are for testing infrastructure and run on dedicated infra-schulcloud Kubernetes clusters, they don't need approval

File: ansible/hosts (hunk 2 above)
Comment (mamutmk5) [reply ansible/hosts:48]: Okay, the 1Password vaulte is put in as secret from the enviorment.
So what exactly are the other differences?

Comment (simoncolincap) [reply ansible/hosts:48]: The ansible configuration is the same at the moment

File: ansible/host_vars/infra.schulcloud-01/version.yml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+SCHULCLOUD_SERVER_IMAGE_TAG: 27.10.0
+SCHULCLOUD_CLIENT_IMAGE_TAG: 27.10.0
+NUXT_CLIENT_IMAGE_TAG: 27.10.0
+SUPERHERO_DASHBOARD_IMAGE_TAG: 27.10.0
+SCHULCLOUD_CALENDAR_IMAGE_TAG: 27.10.0
```
Comment (mamutmk5) [inline ansible/host_vars/infra.schulcloud-01/version.yml:5]:
```suggestion
SCHULCLOUD_SERVER_IMAGE_TAG: 27.10.1
SCHULCLOUD_CLIENT_IMAGE_TAG: 27.10.0
NUXT_CLIENT_IMAGE_TAG: 27.10.1
SUPERHERO_DASHBOARD_IMAGE_TAG: 27.10.1
SCHULCLOUD_CALENDAR_IMAGE_TAG: 27.10.1
```

File: ansible/host_vars/infra.schulcloud-02/version.yml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+SCHULCLOUD_SERVER_IMAGE_TAG: 27.10.0
+SCHULCLOUD_CLIENT_IMAGE_TAG: 27.10.0
+NUXT_CLIENT_IMAGE_TAG: 27.10.0
+SUPERHERO_DASHBOARD_IMAGE_TAG: 27.10.0
+SCHULCLOUD_CALENDAR_IMAGE_TAG: 27.10.0
```
Comment (mamutmk5) [inline ansible/host_vars/infra.schulcloud-02/version.yml:5]:
```suggestion
SCHULCLOUD_SERVER_IMAGE_TAG: 27.10.1
SCHULCLOUD_CLIENT_IMAGE_TAG: 27.10.0
NUXT_CLIENT_IMAGE_TAG: 27.10.1
SUPERHERO_DASHBOARD_IMAGE_TAG: 27.10.1
SCHULCLOUD_CALENDAR_IMAGE_TAG: 27.10.1
```

**Review by mamutmk5 — CHANGES_REQUESTED** [review]

**Review by mamutmk5 — APPROVED** [review]

**Pull request merged**